configure.py

```diff
@@ -633,9 +647,11 @@ def has_re2c():
     print('bootstrap complete.  rebuilding...')
     if platform.is_windows():
         bootstrap_exe = 'ninja.bootstrap.exe'
-        if os.path.exists(bootstrap_exe):
-            os.unlink(bootstrap_exe)
-        os.rename('ninja.exe', bootstrap_exe)
-        subprocess.check_call('ninja.bootstrap.exe%s' % verbose, shell=True)
+        final_exe = 'ninja.exe'
     else:
-        subprocess.check_call('./ninja%s' % verbose, shell=True)
+        bootstrap_exe = './ninja_bootstrap'
+        final_exe = './ninja'
+    if os.path.exists(bootstrap_exe):
+        os.unlink(bootstrap_exe)
+    os.rename(final_exe, bootstrap_exe)
+    subprocess.check_call(bootstrap_exe + verbose, shell=True)
```

Comment:
Is it useful to keep both outputs for possible comparison? Have you ever seen that the bootstrapped exe behaves differently from the final exe?

If not, then I don't think it's any more useful on Win/AIX to keep `ninja_bootstrap` around, since it is always recreated each time you run the bootstrap script. We could auto-delete it on all platforms by adding `os.unlink(bootstrap_exe)` after `subprocess.check_call`, since `check_call` will raise an exception if the process fails.